packages/casting/src/follower-cosmjs.js

```diff
@@ -595,17 +658,17 @@ export const makeCosmjsFollower = (
     async getLatestIterable() {
       return getLatestIterable();
     },
-    async getEachIterable({ height = undefined } = {}) {
-      if (height === undefined) {
-        height = await getBlockHeight();
+    async getEachIterable({ height: blockHeight = undefined } = {}) {
+      if (blockHeight === undefined) {
+        ({ blockHeight } = await getBlockHeight());
```

Comment:
`height` is unambiguous in this context. consider doing the rename in line 663 instead, keeping `height` as the local binding.

even better perhaps would be to have `getBlockHeight` return have `{ height }`. It's obviously _block_ height.